README.md

```diff
@@ -1,169 +1,339 @@
+2. Replace template values with values appropriate to your environment.
+
+### Install
 
-## Configuring the application
+Install NPM Dependencies:
 
-The application receives its configuration from the environment. Each method of launching the app has its own source for the environment. If you run a container image using Kubernetes, it is likely your environment injection will be configured in a Helm chart. For local Docker-based development, you may specifiy the environment or point to an environment file (see Docker documentation for the preferred way to configure this). If running natively using the script included in `package.json`, the app will use either a local `.env` or `.env.dev` file (depending on the script used to launch the app).
+```sh
+  npm install
+```
 
-Environment files are documented [here](./ENVIRONMENT.md), and a sample environment file is provided [here](./env.template).
+### Usage
 
-## Docker compose development environment
+Note: using [docker compose file](docker-compose.yaml) to start the services. This will start the services in development mode.
 
-### Prerequisites
+In order to run the `account-service` in development mode without containers, you can use the following commands:
+
+#### 1. Start the Redis server container, the Frequency container, and the mock webhook server. You can view the logs with your Docker setup.
+
+   ```bash
+   docker compose up -d redis frequency
+   ```
 
-1. Install [Docker Desktop](https://www.docker.com/products/docker-desktop) for your platform
-2. Clone this repository
+#### 2. Once [Frequency](https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9944#/explorer) is up, run the chain setup. This will setup 7000 MSAs on the chain with Graph.
 
    ```bash
-   git clone https://github.com/AmplicaLabs/reconnection-service.git
+   npm run chain-setup
    ```
 
-3. Check for values in `.env.docker.dev` file and update as needed
+   Note: If you change the env's `PROVIDER_ACCOUNT_SEED_PHRASE`, you will also need to update `PROVIDER_ACCOUNT_SEED_PHRASE` in `setup/testing/index.ts`.
 
-### Running the development environment
+#### 3. Start the mock webhook server. You can view the logs with your Docker setup.
 
-Note: Check docker compose file for various services it will start.
-Note: If `graphQueue` is paused, then it means we are out of capacity and need to stake more capacity; in such cases the queue is paused for one epoch and then it will resume automatically.
+  ```bash
+  docker compose up -d webhook
+  ```
 
-1. Start the development environment
+#### 4. Start the Api.<br /><br />
+
+   **Option 1:** In a new terminal window, start the `reconnection-service` app. Logs will be displayed in the terminal for easy reference.
+
+   ```sh
+   npm run start:api:debug
+   ```
+
+   -- or -- <br /><br />
+
+   **Option 2:**
+   Run the following command to start the account service api and worker containers. This will start the account service api and worker in development mode.
```

Comment:
```suggestion
   Run the following command to start the reconnection service container. This will start the reconnection service in development mode.
```